src/components/dragLineLayer.ts

```diff
@@ -0,0 +1,170 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+
+export interface DragLineCallback<D> { (dragLineLayer: Components.DragLineLayer<D>): void; };
+
+export module Components {
+  export class DragLineLayer<D> extends GuideLineLayer<D> {
+    private _dragInteraction: Interactions.Drag;
+    private _detectionRadius = 3;
+    private _detectionEdge: d3.Selection<void>;
+    private _enabled = true;
+
+    private _dragStartCallbacks: Utils.CallbackSet<DragLineCallback<D>>;
+    private _dragCallbacks: Utils.CallbackSet<DragLineCallback<D>>;
+    private _dragEndCallbacks: Utils.CallbackSet<DragLineCallback<D>>;
+    private _disconnectInteractionCallbacks: () => void;
+
+    constructor(orientation: string) {
+      super(orientation);
+      this.addClass("drag-line-layer");
+      this.addClass("enabled");
```

Comment:
But `DragBoxLayer` *does* have a similar implementation:
```CSS
.plottable .drag-box-layer.x-resizable .drag-edge-lr {
  cursor: ew-resize;
}
```
It also sets the CSS classes in a similar way, but in helper methods which make it somewhat less obvious.